utils/run_test.sh

```diff
@@ -43,6 +40,12 @@ fi
 
 run_test()
 {
+    in="$*"
+    a="${in// /-}"
+    b="${a////-}"
```

Comment:
Probably not really necessary to fix here but just as a matter of correctness, function local variables should be defined `local ...`.

Reply:
taking advantage of the transient URL failure to fix and push a new patch